README.md

```diff
@@ -16,65 +16,68 @@
 [travis]: https://travis-ci.org/mcu-tools/mcuboot
 [license]: https://github.com/mcu-tools/mcuboot/blob/main/LICENSE
 
-This is mcuboot version 1.8.0-rc1
+This is MCUboot version 1.8.0-rc1
 
 MCUboot is a secure bootloader for 32-bit MCUs. The goal of MCUboot is to
 define a common infrastructure for the bootloader, system flash layout on
 microcontroller systems, and to provide a secure bootloader that enables
 simple software upgrades.
 
-MCUboot is operating system and hardware independent and relies on
-hardware porting layers from the operating. Currently, mcuboot works
-with both the Apache Mynewt and Zephyr operating systems, but more
-ports are planned in the future. RIOT is currently supported as a boot
-target with a complete port planned.
+MCUboot is operating-system and hardware independent and relies on
+hardware porting layers from the operating system. Currently, MCUboot
+works with the following operating systems:
+- [Zephyr](https://www.zephyrproject.org/)
+- [Apache Mynewt](https://mynewt.apache.org/)
+- [Apache NuttX](https://nuttx.apache.org/)
+- [RIOT](https://www.riot-os.org/)
+- [Mbed OS](https://os.mbed.com/)
+
+RIOT is currently supported only as a boot target. There is currently no
```

Comment:
I explicitly mentioned this because, previously, it was mentioned that there were plans for other ports, but Andrzej just a few messages above pointed out that this is no more the case. Sure, it can be removed if we don't want to explicitly state that.

Reply:
Rephrased.